Cargo.toml

```diff
@@ -36,7 +36,7 @@ __doctest = []
 
 [dependencies]
 libc = { version = "0.2.69", optional = true }
-time = { version = "0.1.43", optional = true }
+time = { version = "0.3", optional = true }
```

Comment:
0.3.4 was recently released, perhaps use it here?

Reply:
```suggestion
time = { version = "0.3.4", optional = true }
```

Reply:
There's no reason to bump a minor version unless it is actually required for compiling.

Reply:
Odd, my local rust was complaining about something. Shall look into it.

Reply:
Oh jeez, my bad. A branch didn't get cut off properly, ignore this.